checkov/terraform/checks/provider/aws/defaulttags.py

```diff
@@ -0,0 +1,28 @@
+import re
+from typing import Dict, List, Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.provider.base_check import BaseProviderCheck
+
+
+class AWSDefaultTags(BaseProviderCheck):
+    def __init__(self) -> None:
+        name = "AWS provider should specify default tags"
+        id = "CKV_AWS_166"
+        supported_provider = ["aws"]
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
just a general question, wouldn't `CONVENTION` fit better here or does it make sense to even create a new category like `BEST_PRACTICE`?

Reply:
Interesting idea. WDYT @schosterbarak @GiladMark

Reply:
CONVENTION sounds good